package.json

```diff
@@ -14,6 +14,160 @@
+    "publish": "cd tmp && npm publish && cd ..",
+    "publish-all": "npm run all publish"
+  },
+  "keywords": [
+    "data",
+    "grid",
+    "table",
+    "editor",
+    "grid-editor",
+    "data-grid",
+    "data-table",
+    "spreadsheet",
+    "excel",
+    "tabular-data",
+    "edit-cell",
+    "editable-table",
+    "data-spreadsheet"
+  ],
+  "dependencies": {
+    "@types/pikaday": "1.7.4",
+    "core-js": "^3.0.0",
+    "dompurify": "^2.1.1",
+    "hot-formula-parser": "^4.0.0",
+    "moment": "2.24.0",
+    "numbro": "2.1.2",
+    "pikaday": "1.8.0"
+  },
+  "devDependencies": {
+    "@actions/core": "^1.2.6",
+    "@actions/github": "^4.0.0",
+    "@babel/cli": "^7.8.3",
+    "@babel/core": "^7.11.4",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
+    "@babel/plugin-transform-runtime": "^7.11.0",
+    "@babel/polyfill": "^7.8.3",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/register": "^7.8.3",
+    "@babel/runtime": "^7.11.2",
+    "@babel/types": "^7.12.12",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-eslint": "^11.0.0-beta.2",
+    "babel-jest": "^26.6.3",
+    "babel-loader": "^8.0.4",
+    "babel-plugin-forbidden-imports": "^0.1.2",
+    "babel-plugin-transform-inline-environment-variables": "^0.4.3",
+    "babel-plugin-transform-require-ignore": "^0.1.1",
+    "chalk": "^4.1.0",
+    "concurrently": "^5.3.0",
+    "copy-webpack-plugin": "^6.0.3",
+    "cpy-cli": "^3.1.1",
+    "cross-env": "^5.0.1",
+    "css-loader": "^2.1.1",
+    "ecstatic": "^4.1.4",
+    "env-cmd": "^9.0.3",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-babel": "^5.3.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsdoc": "^20.3.1",
+    "execa": "^5.0.0",
+    "fs-extra": "^4.0.2",
+    "glob": "^7.1.2",
+    "html-parse-stringify": "^1.0.3",
+    "html-webpack-plugin": "^3.2.0",
+    "inquirer": "^7.3.3",
+    "jasmine-co": "^1.2.2",
+    "jasmine-core": "^3.4.0",
+    "jasmine-terminal-reporter": "^1.0.3",
+    "jest": "^26.6.3",
+    "jest-cli": "^26.6.3",
+    "jsdom": "^15.2.1",
+    "jsdom-global": "^3.0.2",
+    "loader-utils": "^1.1.0",
+    "lodash": "^4.17.20",
+    "mini-css-extract-plugin": "^0.5.0",
+    "on-build-webpack": "^0.1.0",
+    "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "progress-bar-webpack-plugin": "^1.10.0",
+    "puppeteer": "^2.1.1",
+    "replace-in-file": "^6.1.0",
+    "rimraf": "^2.5.4",
+    "spawn-command": "0.0.2",
+    "string-replace-webpack-plugin": "^0.1.3",
+    "style-loader": "^0.18.2",
+    "stylelint": "^13.3.3",
+    "supports-color": "^4.2.1",
+    "tree-kill": "^1.2.2",
+    "typescript": "3.8.2",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.0",
+    "yargs": "^16.2.0"
+  },
+  "license": "SEE LICENSE IN LICENSE.txt",
+  "resolutions": {
+    "babel-core": "7.0.0-bridge.0"
+  },
+  "workspaces": {
+    "packages": [
+      ".",
+      "wrappers/*"
```

Comment:
What do you think about changing "wrappers" to "packages"? What we will do when another project appears in the project and it is not a wrapper. We will create another directory and add it to the "packages" array?

Reply:
Yeah, that's yet another thing to discuss. 
I think we (@wojciechczerniak and me) decided to go with `wrappers` because it clearly distinguishes itself from the rest of the repo. If it was a part of a `packages` directory, among other future projects like `examples` etc, it would be harder to find just the wrappers.
We can always go with a mixed approach, with something like:
- `.` -> `handsontable`
- `./packages/wrappers/**` -> wrappers
- `./packages/examples/` -> examples (in the future)

etc.

What do you think, @budnix, @wojciechczerniak?

Reply:
I'd stick to the idea of the flat structure. Something like this:
![image](https://user-images.githubusercontent.com/571316/106256140-318e9900-621b-11eb-9b8e-6ccb73846954.png)

In the context of the Handsontable, it's known what the package does.

Reply:
`handsontable` will still not be in the `packages` directory, so it wouldn't be a flat-flat structure. But yeah, I'm open for suggestions.

Reply:
I know it looks tempting to just rename `wrappers` to `packages`, but soon it will look very different. There will be few directories that are not packages, but will have dependencies on them. We will add more directories to the structure soon:
- `/docs` - will use `handsontable` to generate API reference, and should be on the main level, always visible and accessible
- `/examples` - will use `handsontable` and wrappers packages, but are not packages and will not be published
- `renderes`, `editors`, `validators` are part of the `/src/` and will not be moved to `/packages` so, in the end, we will end up with a mix and misleading `/packages` directory that should consolidate all of them, but is not
Just to name few problems with `/packages`. `/packages` is a convention from Lerna. I've seen monorepos that were using it at first and then moving the files to `/wrappers` and `/examples`, losing all the GIT history again. 

TL;DR; IMO we should keep the current structure idea.